src/material/core/option/option.html

```diff
@@ -1,19 +1,18 @@
-<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox"
-    [state]="selected ? 'checked' : 'unchecked'" [disabled]="disabled"></mat-pseudo-checkbox>
+<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox" [disabled]="disabled"
```

Comment:
It would be good to add a comment in this file on why `aria-hidden` is necessary, since most non-role, non-text-content elements are already a no-op for assistive technology